### PR TITLE
Remap component library imports

### PR DESCRIPTION
**Pull request by bkasten-rbi**
wip

### CODE REVIEW

File: codemods/components-lib-imports/mapping.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
+export const mapping = {
+  // Basic remaps that should _just work_
+  Box: { name: 'Box', import: '@rbilabs/universal-components' },
+  VisuallyHidden: { name: 'VisuallyHidden', import: 'components/ucl' },
+  TextInput: { name: 'TextInput', import: 'components/ucl/text-input' },
+  ITextInputProps: { name: 'TextInputProps', import: 'components/ucl/text-input/types' },
+  Select: { name: 'Select', import: '@rbilabs/universal-components' },
+  Checkbox: { name: 'Checkbox', import: '@rbilabs/universal-components' },
+  useMediaQuery: { name: 'useMediaQuery', import: 'hooks/use-media-query', defaultImport: true },
+  NumberInput: { name: 'NumberInput', import: 'components/ucl/number-input' },
```
Comment (kfinley-rbi) [inline codemods/components-lib-imports/mapping.ts:10]: This one is also in the index
```suggestion
  NumberInput: { name: 'NumberInput', import: 'components/ucl' },
```

File: codemods/components-lib-imports/mapping.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  // this is basically a Badge with the correct icon per brand and a variant="default-solid"
+  // i think we could build this quick in UCL and import it
+
+  // Kind of sure, but not totally sure if this is right.
+  Toggle: { name: 'Switch', import: '@rbilabs/universal-components' },
+  IToggleOption: { create: `type ISwitchOption = React.ComponentProps<typeof Switch>` },
+
+  // Not sure yet... where do these remap to??
+  RadioButton: false,
+  OutlineButton: false,
```
[the remark this answers is not in the document]
Comment (bkasten-rbi) [reply codemods/components-lib-imports/mapping.ts:55]: i'm going to do a PR actually now to just delete all .ltw files

File: codemods/components-lib-imports/mapping.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  TextOnlyButton: false,
+  IFooterProps: false,
+  InlineAlert: false,
+  ThemeColorArg: false,
+  getThemeColor: false,
+  TextArea: false,
```
Comment (kfinley-rbi) [inline codemods/components-lib-imports/mapping.ts:61]: Once used once. 
There's UCL `TextArea` and formik version `TextArea`

File: codemods/components-lib-imports/mapping.ts [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+  ITextAreaProps: false,
```
Comment (kfinley-rbi) [inline codemods/components-lib-imports/mapping.ts:62]: I don't think we are exporting this from the UCL, but we can

File: codemods/components-lib-imports/mapping.ts [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  MenuCarouselItemProps: false,
+  useMenuCarouselHandler: false,
+
+  // Not sure yet.. but i think we skip these ones:
+  useDialogUIState: false, // only used in one file. I think we should just do this one manually
+  LocaleSelect: false, // Only used in one file. Maybe we skip that file.
+  IIcon: false, // Its used to get the `keyof IIcon` e.g., a type of all the Icon names
+  Footer: false, // Does it apply to mobile usage?
+
+  // drop these imports
```
Comment (kfinley-rbi) [inline codemods/components-lib-imports/mapping.ts:74]: 👍

File: codemods/components-lib-imports/mapping.ts [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+  MenuCarousel: false,
+  MenuCarouselItem: false,
+  MenuCarouselItemProps: false,
+  useMenuCarouselHandler: false,
+
+  // Not sure yet.. but i think we skip these ones:
+  useDialogUIState: false, // only used in one file. I think we should just do this one manually
+  LocaleSelect: false, // Only used in one file. Maybe we skip that file.
+  IIcon: false, // Its used to get the `keyof IIcon` e.g., a type of all the Icon names
+  Footer: false, // Does it apply to mobile usage?
```
Comment (kfinley-rbi) [inline codemods/components-lib-imports/mapping.ts:72]: We'll have to have someone convert this manually, it's big... Or we could try running the code mod against the `components-library` 😄 ?

File: codemods/components-lib-imports/mapping.ts [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+  IFooterProps: false,
+  InlineAlert: false,
+  ThemeColorArg: false,
+  getThemeColor: false,
+  TextArea: false,
+  ITextAreaProps: false,
+  MenuCarousel: false,
+  MenuCarouselItem: false,
+  MenuCarouselItemProps: false,
+  useMenuCarouselHandler: false,
```
Comment (kfinley-rbi) [inline codemods/components-lib-imports/mapping.ts:66]: Might be better to do these manually too, we are still converting the UI

**Review by kfinley-rbi — COMMENTED** [review]
This looks awesome!

File: codemods/components-lib-imports/mapping.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (kfinley-rbi) [reply codemods/components-lib-imports/mapping.ts:55]: > i'm going to do a PR actually now to just delete all .ltw files
🍾

File: codemods/components-lib-imports/mapping.ts (hunk 6 above)
Comment (bkasten-rbi) [reply codemods/components-lib-imports/mapping.ts]: yeah im skipping that one haha

**Pull request merged**